INSTALLING.md

```diff
@@ -56,23 +56,62 @@ sudo -E python3 setup_hugepages.py enable && sudo -E python3 setup_hugepages.py
 ```
 ---
 
-### Step 4. Build from source and start using TT-NN and TT-Metalium!
+### Step 4. Clone tt-metal repo
```

Comment:
Are the steps before this one, and particularly step 1, still required for the Docker path? I see Install "tt-smi/tt-flash/tt-topology" in the Dockerfile so want to make sure

Reply:
So the installation Hugepages (Step 3) and tt-kmd drivers must be installed on the host side as this can't be done in docker. The firmware can be flashed using docker too.

In essence, half of Step 1 and Step 3 are required, and Step 2 is not required if developing in docker.

I think a split into two files is probably a better choice here

Reply:
Thanks for clarifying @ttmchiou and agree with that. We could link the Docker installation steps right at the beginning of this file by adding something similar to what I share here in Italic (please feel free to rephrase):

"These instructions will guide you through the installation of Tenstorrent system tools and drivers, followed by the installation of TT-Metalium and TT-NN.

_The following steps will help you configure a native host development environment. If you prefer to work in a Docker container, please follow this guide: <link_to_docker_steps>._"

Reply:
@adifrancescoTT Do you think its better to have it split into two files or just two independent sections but still living in INSTALLING.md?
Option 1: INSTALLING.md + DOCKER_INSTALLING.md
or 
Option 2: INSTALLING.md
but it would look something \
# Intro
for native developemnt, go here <link to Native Developement section>
for docker development, go here <link to Docker Development section>

## Native Development
Instructions

## Docker Development
Instructions

Reply:
Not a super strong preference, but I think I'm leaning towards what you're suggesting (option 2) so go with it if you're also aligned. My main ask is that each section is fully "self-contained", meaning a user can follow it from start to end and achieve their goal without referencing the other one. Thanks!

Reply:
Not a super strong preference, but I think I'm leaning towards what you're suggesting (option 2) so go with it if you're also aligned. My main ask is that each section is fully "self-contained", meaning a user can follow it from start to end and achieve their goal without referencing the other one. Thanks!

Reply:
@adifrancescoTT 
Updated, let me know your thoughts